operate/docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
       - elasticsearch
 
   opensearch:
-    image: opensearchproject/opensearch:2.13.0
+    image: opensearchproject/opensearch:2.5.0
```

Comment:
Does it make a difference if we use newer (2.13.0) dependency here? ie. on docs we state we support 2.5+ so would this potentially break because of the java client and that is reason for downgrading back to 2.5.0?

Reply:
Thanks! My bad. I recognized it as 2.1. Will remove the commit.